fhir-persistence-schema/README.md

```diff
@@ -321,8 +325,8 @@ java -jar ./fhir-persistence-schema-%VERSION%-cli.jar --to-file
 Note: the jar file is stored locally in `fhir-persistence-schema/target` or in the Artifactory repository for this project.
 
 2 - Connect to your instance and execute each of the following:
-    - schema.sql:  `db2 -tvf schema.sql`
-    - grants.sql:  `db2 -tvf grants.sql`
-    - stored-procedures.sql:  `db2 -td@ -vf stored-procedures.sql`
+ * schema.sql:  `db2 -tvf schema.sql`
+ * grants.sql:  `db2 -tvf grants.sql`
+ * stored-procedures.sql:  `db2 -td@ -vf stored-procedures.sql`
```

Comment:
we just need a return from 327 to 328, I'd opt for that change.

Reply:
changed as requested; I thought, an enumeration was intended, not a code block (4 spaces...)